BrainPortal/app/controllers/neurohub_application_controller.rb

```diff
@@ -122,5 +97,13 @@ def prepare_invites
     @nh_new_invites_count = nh_new_invites.count
   end
 
+  # this patches prepare_message method
+  # to display 'Invitation Accepted' entitled messages. They will be show separately from Invitations
+  # and only kind of message to be shown for now.
+
```

Comment:
You must not leave a blank line between the method description and the method, otherwise the documetnation code produces a method with no description

Reply:
I'd also suggest adding the #:nodoc: keyword because that method is not really a method that CBRAIN developer will want to invoke at other places.

Reply:
Same with the copy in ApplicationController

Reply:
nodoc added